### PR TITLE
feat(project) Create project

### PR DESCRIPTION
**Pull request by sleclercq**
name is mandatory. Added namespace_id to specify namespace (gitlab defaults to user's own namespace).

### CODE REVIEW

File: projects_test.go [hunk 1]
```diff
@@ -26,6 +26,18 @@ func TestProject(t *testing.T) {
...
+func TestCreateProject(t *testing.T) {
+	ts, gitlab := Stub("stubs/projects/show.json")
+	project := Project{
+		Name: "Diaspora Project Site",
+	}
+
+	result, err := gitlab.CreateProject(&project)
+	assert.NoError(t, err)
+	assert.Equal(t, project.Name, result.Name)
+	defer ts.Close()
```
Comment (rande) [inline projects_test.go]: defer should be after `ts` declaration.

File: examples/projects/main.go [hunk 2]
```diff
@@ -91,53 +149,62 @@ func main() {
...
+			project := buildProject(name, namespaceId, desc, builds, mergeRequests)
+
+			result, err := gitlab.CreateProject(&project)
+
+			if err != nil {
+				fmt.Println(err.Error())
+				return
+			}
+
+			printProject(*result)
```
Comment (rande) [inline examples/projects/main.go]: Why are you dereferencing the pointer ?

**Comment by plouc** [comment]
@sleclercq, do you plan to finalize this PR ?

**Comment by sleclercq** [comment]
I have no need for it anymore so unless you find it useful I have no plans for it.

**Pull request closed without merging**